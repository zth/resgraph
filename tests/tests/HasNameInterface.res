/** An entity with a name. */
@gql.interface
type hasName = {@gql.field name: string}
//    ^hov
