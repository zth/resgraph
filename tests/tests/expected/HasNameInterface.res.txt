Hover tests/HasNameInterface.res 2:6
{"status": "Hover", "item": {"contents": {"kind": "markdown", "value": "GraphQL interface generated by ResGraph:\n```graphql\n\n\"\"\"An entity with a name.\"\"\"\ninterface HasName {\n  abbreviatedName: String\n  name: String!\n}\n```"}}}
